### PR TITLE
Math command

### PR DESCRIPTION
**Pull request by bjornmorten**

### CODE REVIEW

**Comment by briantanner** [comment]
Missing semicolons everywhere

**Comment by bjornmorten** [comment]
Added semicolons now

File: src/commands/Math.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		this.expectedArgs = 1;
+	}
+
+	execute({ message, args }) {
+		try {
+			let input = args.join(' ');
+            		let output = math.eval(input).toString();
+            		return this.sendMessage(messsage.channel, `Answer: ${output}`);
+		} catch (e) {
+			return this.sendMessage(message.channel, "Couldn't evaluate the given expression!");
```
Comment (germanoeich) [inline src/commands/Math.js:23]: use `this.error(channel, message)` instead of `this.sendMessage`

Also use single quotes for strings

**Review by germanoeich — CHANGES_REQUESTED** [review]

File: src/commands/Math.js (hunk 1 above)
Comment (germanoeich) [reply src/commands/Math.js]: Also single quotes please

**Pull request closed without merging**